lib/modules/model/types/ModelContent.ts

```diff
@@ -33,18 +33,30 @@ export interface LocaleDetails {
   description: string;
 }
 
+interface DefinitionMetadataDetails {
+  readOnly?: boolean;
+  type?: string;
+  values?: string[] | number[] | boolean[];
+  customValueAllowed?: boolean;
+}
+
+interface DatetimeMetadataDetails {
+  date?: boolean;
+  time?: boolean;
+  timezone?: boolean;
```

Comment:
What happens when timezone is false ? We don't ask for a timezone ?
If that's the case, what timezone will be inferred ? Will we process date and times as UTC or as localtime ? Client or server localtime ?

Reply:
date shouldn't optional in a Datetime picker

Reply:
Yes I understand now that timezone must be always informed to avoid problems through your questionings. I will remove the property and set date to mandatory